src/api/entities/confidential/ConfidentialAsset/types.ts

```diff
@@ -16,3 +16,26 @@ export interface GroupedAuditors {
   auditors: ConfidentialAccount[];
   mediators: Identity[];
 }
+
+export interface CreateConfidentialAssetParams {
+  /**
+   * Custom data to be associated with the confidential Asset
```

Comment:
```suggestion
   * custom data to be associated with the confidential Asset
```

feels inconsistent when "optional" is small case on other lines. Could capitalize "Optional" on 26 and 34 instead